bugbug/tools/code_review.py

```diff
@@ -62,16 +62,29 @@ class HunkNotInPatchError(ModelResultError):
 
 {patch}"""
 
-PROMPT_TEMPLATE_REVIEW = """You will be given a task for generate a code review for the patch below. Use the following steps to solve it.
-
-{patch}
-
+PROMPT_TEMPLATE_REVIEW = """You will be given a task for generate a code review for the patch below. Use the following steps to solve it:
```

Comment:
```suggestion
PROMPT_TEMPLATE_REVIEW = """You will be given a task for generating a code review for the patch below. Use the following steps to solve it:
```

Reply:
Nice catch! The typo is out of the scope of this PR, but I fixed it in 3aea837c99613eac22e3c859499f667278409515 since we are touching this line.